### PR TITLE
add user_role and dob

### PR DESCRIPTION
**Pull request by abhishekanalytics**

### CODE REVIEW

File: Flask_app/__init__.py [hunk 1]
```diff
@@ -1,19 +1,26 @@
 import os
-from flask import Flask 
+
+from flask import Flask
 from flask_sqlalchemy import SQLAlchemy
-from .config import config
 
- 
+from Flask_app.config import config
```
Comment (aayush-kumar-codes) [inline Flask_app/__init__.py]: Directory name should be capital and each word should be separated by underscore

File: Flask_app/models/models.py [hunk 2]
```diff
@@ -1,12 +1,22 @@
...
+
+
 from werkzeug.security import(
     generate_password_hash,
     check_password_hash
     )
 
 
+
+class user_role(Enum):
```
Comment (aayush-kumar-codes) [inline Flask_app/models/models.py]: Class name conventionally wrong, It should be like UserRole

File: Flask_app/models/models.py [hunk 3]
```diff
@@ -19,7 +29,9 @@ class User(db.Model):
     lastname=Column(String(20),nullable=False)
     mobile_number=Column(String(20),nullable=False)
     hash_password = Column('password', String(350), nullable=False)
-    age = db.Column(db.Integer)
+    date_of_birth = Column(Date, nullable=True)
```
Comment (pythexcel) [inline Flask_app/models/models.py:32]: No property added for calculating age

File: Flask_app/views/user.py [hunk 4]
```diff
@@ -1,5 +1,5 @@
 from flask import jsonify,Blueprint
-admin = Blueprint('admin', __name__, url_prefix='/admin')
-@admin.route('/')
+bp = Blueprint('bp', __name__, url_prefix='/users')
```
Comment (aayush-kumar-codes) [inline Flask_app/views/user.py]:
```suggestion
bp = Blueprint('bp', __name__, url_prefix='/users')
```

user_bp = Blueprint('user_bp', __name__, url_prefix='/users')

**Pull request merged**